kymatio/scattering3d/frontend/torch_frontend.py

```diff
@@ -8,6 +8,40 @@
 
 
 class HarmonicScatteringTorch3D(ScatteringTorch, ScatteringBase3D):
+    """3D Solid Harmonic scattering .
```

Comment:
Why is all this here? Doesn't it get auto-documented using the `_document` class method?

Reply:
My bad,  I didn't bother removing this during the cherry-pick